ebu_tt_live/node/denester.py

```diff
@@ -575,17 +575,17 @@ def create_new_style(new_style, dataset):
         the name are the same.
         If not, a new style is created and added to the dataset.
         """
-        if dataset["styling"] is None:
+        styling = dataset["document"].head.styling
```

Comment:
doesn't this break line 581 by making a more locally scoped thing called `styling` that is no longer the styling type?